### PR TITLE
Update concept sets for UC1 tables 5.15 - 5.19 & UC3 tables 

### PR DESCRIPTION
**Pull request by LP-Glueck**
Variables from UC3 have been added as csv files to phems_variables_list directory also in this PR.

I still need to finish the drug mappings as we don't have all the possible mappings specified for the Factor VIII treatment variable (There can be potentially a lot). I will check if descendants can be used at all, but I think I will have to specify all possible drug mappings that we gave them as options because they are not subsumes of each other.

### CODE REVIEW

File: project/R/conceptSets.R [hunk 1]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
       4019026, # source: Other specified: plastic repair of aorta /// concept: Plastic repair of aorta
       44510968  # source: Other specified other operations on ventricles of heart /// concept: Other specified other operations on ventricles of heart
+    ),
+    
+    vitalSigns = c(
+      3027018, # source: Age-dependent HR /// concept: Heart rate
+      3024171, # source: Age-dependent RR /// concept: Respiratory rate
+      3004249, # source: SBP (Systolic Blood Pressure) /// concept: Systolic blood pressure
+      3012888, # source: DBP (Dyastolic Blood Pressure) /// concept: Diastolic blood pressure
+      4096101 # source: SpO2 /// concept: Measurement of oxygen saturation at periphery
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: Maybe a better concept: [Oxygen saturation measurement](https://athena.ohdsi.org/search-terms/terms/4020553)

File: project/R/conceptSets.R [hunk 2]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      4201235, # source: Body Surface Area (BSA) - Combination of weight and height /// concept: body surface area
+      4275564 # source: Cardiac surgeries (Interventions) /// concept: Operation on heart
+    ), 
+    
+    HSJDSurgery = c(
+      40490494, # source: STAT/EACTS score of surgery /// concept: Society of Thoracic Surgeons risk calculator
+      4301351, # source: Surgery start datetime, end datetime /// concept: Surgical procedure
+      4336464, # source: Cardiac bypass start datetime, end datetime /// concept: Coronary artery bypass graft
+      4201547, # source: Cross-clamping start datetime, end datetime /// concept: Placement of arterial cross clamp
+      44790138, # source: Deep hypothermic circulatory arrest start datetime, end datetime /// concept: Induced circulatory arrest
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: Maybe a better concept: [Hypothermic circulatory arrest](https://athena.ohdsi.org/search-terms/terms/37152781)

File: project/R/conceptSets.R [hunk 3]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      4272324, # source: Antegrade Cerebral Perfusion start datetime, end datetime /// concept: Cardiopulmonary bypass operation
+      4150627 # source: Extubation event /// concept: Removal of endotracheal tube
+    ),
+    
+    HSJDHospitalisation = c(
+      44803020, # source: Admission reason /// concept: Primary reason for admission
+      42539650, # source: Vasoactive infusion start datetime, end datetime /// concept: Administration of intravenous vasoactive drug
+      794078, # source: Dose of vasoactive infusion (rate - mcg/kg/hr) /// concept: vasoactive intestinal peptide
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: This doesn't need to be a concept but can be derived from the dose field of drug exposure record for the vasoactive drug.

File: project/R/conceptSets.R [hunk 4]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      3011832, # source: factor VIII activity measurement /// concept: coagulation factor VIII activity [Units/volume] in platelet poor plasma by Chromogenic assay
+      43534000, # source: Von Willebrand factor activity measurement /// concept: von Willebrand factor (vWf) activity [Units/volume] in platelet poor plasma by Immunoassay
+      3002124, # source: Von Willebrand factor antigen measurement /// concept: von Willebrand factor (vWf) Ag [Units/volume] in platelet poor plasma by Immunoassay
+      3023693, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) multimers in platelet poor plasma by Immunoblot
+      3042349, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) cleaving protease inhibitor [Units/volume] in platelet poor plasma
+      4175016, # source: activated partial thromboplastin time (APTT) /// concept: partial thromboplastin time, activated
+      3034426, # source: Prothrombin time (PT) /// concept: prothrombin time (PT)
+      37393863, # source: platelet count /// concept: platelet count
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: This is a not standard concept. The standard one is: [Platelet count](https://athena.ohdsi.org/search-terms/terms/4267147)

File: project/R/conceptSets.R [hunk 5]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      40492862, # source: factor VIII dose is continuous infusion or bolus dose /// concept: mode of drug administration
+      4129275, # source: factor VIII dose is continuous infusion or bolus dose /// concept: continuous infusion
+      4265597, # source: factor VIII dose is continuous infusion or bolus dose /// concept: by bolus infusions
+      1517070 # source: desmopressin /// concept: desmopressin
+      # TODO FIGURE OUT IF DESCENDENTS WORK WITH THESE CONCEPTS 
+      # TODO MAP REMAINING DRUG BRANDS 
+    ),
+    
+    UC3Surgery = c(
+      45888085, # source: Medical/ surgical procedure /// concept: medical procedure
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: This concept is a CPT4 classification code for [Surgery](https://athena.ohdsi.org/search-terms/terms/45888085). Use instead [Therapeutic procedure](https://athena.ohdsi.org/search-terms/terms/4172515)

File: project/R/conceptSets.R [hunk 6]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      967823, # source: NaCl administration during surgery /// concept: sodium chloride
+      4028665, # source: plasma administration during surgery /// concept: plasma transfusion
+      45888867, # source: datetime start anesthesia /// concept: anesthesia
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: This concept is also a CPT4 classification code. Use instead [Administration of anesthesia](https://athena.ohdsi.org/search-terms/terms/4160439)

File: project/R/conceptSets.R [hunk 7]
```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
...
+      4308716, # source: blood loss during surgery /// concept: intraoperative hemorrhage
+      37017589 # source: blood transfusion during surgery /// concept: bleeding during medical procedure requiring transfusion
```
Comment (SofiaMp) [inline project/R/conceptSets.R]: The concept name is slightly different [Bleeding during surgery requiring transfusion](https://athena.ohdsi.org/search-terms/terms/37017589)

**Review by SofiaMp — CHANGES_REQUESTED** [review]
Found some concepts that can change. Only minor things. I understand that the csv files would need to be updated but I did not add the comments there.

**Comment by LP-Glueck** [comment]
Ready for review with updated changes 👍🏼

**Comment by guuswilmink** [comment]
Please add some comments to the conceptSets to indicate the source table from D4.1
For example for vital signs;
```
#  5.22 Vital Signs
vitalSigns = c(
  ...  # source: <> \\\ concept: <>
)
```

Other than that looks good! You can merge after this change

**Review by guuswilmink — APPROVED** [review]

**Pull request merged**